scripts/pre-release.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```

Comment:
I am not sure about this line

Reply:
Is it required to do this for a functional reason?

Reply:
my wrong. I added it by mistake. I've been using it for my testing on my local machine.